test/unit/test_credentials_holder.py

```diff
@@ -57,9 +74,54 @@ def test_aws_profile_credentials_holder_should_have_session():
 
 def test_aws_profile_credentials_holder_get_session_credentials():
     profile_val: str = "myprofile"
-    obj: AWSProfileCredentialsHolder = AWSProfileCredentialsHolder(profile=profile_val, session=MagicMock())
+    obj: AWSProfileCredentialsHolder = AWSProfileCredentialsHolder(
+        profile=profile_val, session=MagicMock()
+    )
 
     ret_value = obj.get_session_credentials()
     assert len(ret_value) == 1
 
     assert ret_value["profile"] == profile_val
+
+
+@pytest.mark.parametrize(
+    "expiration",
+    [
+        datetime.datetime.now() - datetime.timedelta(hours=3),  # expired 3 hrs ago
+        datetime.datetime.now() - datetime.timedelta(days=1),  # expired 1 day ago
+        datetime.datetime.now() - datetime.timedelta(weeks=1),  # expired 1 week ago
+    ],
+)
+def test_is_expired_true(expiration):
+    credentials: typing.Dict[str, typing.Any] = {
+        "AccessKeyId": "something",
+        "SecretAccessKey": "secret",
+        "SessionToken": "fornow",
+        "Expiration": expiration.astimezone(),
```

Comment:
The Expiration we receieve from boto3 is a datetime with UTC timezone.  I think we could replace `datetime.datetime.now()` with `datetime.now(timezone.utc)` to achieve that. This would allow us to directly use `expiration` when creating the `credentials`dict